src/nidmm/examples/nidmm_multi_point_measurement.py

```diff
@@ -16,7 +16,7 @@ def example(argsv):
     parser.add_argument('-op', '--option_string', default='', type=str, help='Option string')
     args = parser.parse_args(argsv)
 
-    with nidmm.Session(resource_name=args.resource_name, option_string=args.option_string) as session:
+    with nidmm.Session(resource_name=args.resource_name, options=args.option_string) as session:
```

Comment:
1. It's a shame our examples don't show the "preferred" way of using this.
2. Also, when running these from the system tests, we won't be exercising the dictionary->string code.

The second one can be fixed with a little bit more example refactoring. Specifically, the example() method should take specific parameters rather than argsv. But then we aren't testing the command line argument parsing... so we would have to invoke both ways? Not sure what the best way to proceed is.

Reply:
I have modified the nidcpower examples to use and test both an option string and an option dictionary. Let me know if you think the added complexity and noise in an example is worth it to show both ways of passing in options.